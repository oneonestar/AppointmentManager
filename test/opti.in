addStudy -alice 2015-08-04 18:00 3.0
addAssignment -bob 2015-08-04 18:00 3.0
addGathering -charlie 2015-08-04 19:00 2.5
addAssignment -alice 2015-08-04 20:00 3.0
addStudy -bob 2015-08-04 20:00 3.0
addStudy -charlie 2015-08-04 22:00 3.0
addProject -alice 2015-08-04 17:00 2 bob
addGathering -bob 2015-08-04 17:00 3.0 alice charlie
printSchd -opti
