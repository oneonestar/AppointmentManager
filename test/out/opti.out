~~WELCOME TO AMR~~
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
-> [Pending]
Please enter appointment:
***Appointment Schedule – ACCEPTED ***

Alice, you have 4 appointments.
 3 2015-08-04   20:00   23:30   Assignment   N         -
 6 2015-08-05   18:00   20:30   Project      Y         Alice Bob 
 7 2015-08-06   18:00   21:30   Gathering    Y         Bob Alice Charlie 
 0 2015-08-07   18:00   21:30   Study        Y         -
Bob, you have 4 appointments.
 1 2015-08-04   18:00   21:30   Assignment   N         -
 6 2015-08-05   18:00   20:30   Project      Y         Alice Bob 
 7 2015-08-06   18:00   21:30   Gathering    Y         Bob Alice Charlie 
 4 2015-08-07   18:00   21:30   Study        Y         -
Charlie, you have 3 appointments.
 2 2015-08-04   19:00   21:00   Gathering    N         -
 5 2015-08-04   22:00   01:30   Study        N         -
 7 2015-08-06   18:00   21:30   Gathering    Y         Bob Alice Charlie 
   -End-
===========================================================================
***Appointment Schedule – REJECTED ***

Alice, you have 0 appointments rejected.
Bob, you have 0 appointments rejected.
Charlie, you have 0 appointments rejected.
   -End-
===========================================================================
Performance:
Date start: 2015-08-04
Date end: 2015-08-05

Total Number of Appointment Assigned: 8 (100.0%)
Total Number of Appointment Rejected: 0 (0.0%)
Utilization of Time Slot: (3 days)
    Alice      - 56.2%
    Bob        - 75.0%
    Charlie    - 25.0%
Please enter appointment:
Received EOF.